scripts/docker/Dockerfile

```diff
@@ -1,27 +1,28 @@
-FROM ubuntu:xenial
+FROM debian:stretch-slim
 LABEL maintainer "devops-team@parity.io"
 LABEL description="Substrate: The platform for blockchain innovators"
 
 RUN apt-get update && \
-    apt-get upgrade -y && \
+    DEBIAN_FRONTEND=noninteractive apt-get upgrade -y && \
     DEBIAN_FRONTEND=noninteractive apt-get install -y \
-		libssl1.0.0 \
-		ca-certificates \
-		curl && \
+      libssl1.1 \
+      ca-certificates \
+      curl && \
```

Comment:
is `curl` needed?

Reply:
maybe not, I don't know, but generally useful (I don't mind removing it)